packages/fxa-admin-panel/src/components/AccountSearch/Account/index.tsx

```diff
@@ -431,17 +416,40 @@ export const Account = ({
               ))}
             </>
           ) : (
-            <li
-              className={`${styleClasses.borderInfoDisplay} ${styleClasses.li}`}
-            >
+            <li className="account-li account-border-info">
               This account has nothing attached.
             </li>
           )}
         </Guard>
+
+        {/* Temporary check for fake hard-coded value until we fetch actual subscriptions in FXA-4237 */}
+        {subscriptions &&
+          subscriptions[0].productName !== 'Cooking with Foxkeh' && (
```

Comment:
I could just check against the ENV too, but it doesn't matter too much since I'm ripping this out with my next ticket anyway

Reply:
This seems fine to me, and in some ways preferable since env checks have a tendency to linger.